src/main/java/org/fcrepo/migration/handlers/ocfl/PlainOcflObjectSession.java

```diff
@@ -144,6 +164,12 @@ public synchronized void commit() {
                     } else {
                         updater.addPath(objectStaging, ocflOptions);
                     }
+                    digests.forEach((contentPath, digestInfo) -> {
+                        digestInfo.forEach((digestType, digestValue) -> {
+                            updater.addFileFixity(contentPath, DigestAlgorithm.fromOcflName(digestType), digestValue);
```

Comment:
Sorry I didn't notice this earlier, but this path will not be correct on Windows. `contentPath` here is the percent-encoded path that was created to safely write the file to the staging directory. The path that you need to pass `addFileFixity` is the OCFL logical path of the file. Have a look at `addDecodedPaths` for one possible solution to this problem.

Reply:
Np. I updated it, to use the logical path before encode() is called. Does that fix the issue?

Reply:
Yes, I believe that should be correct.

Just need to try it locally and I think you should be good to go.